### PR TITLE
Fix ACME HTTP metrics names

### PR DESCRIPTION
**Pull request by arapulido**
### What does this PR do?

Cert-Manager ACME related metrics changed their name in the latest versions. This PR uses the new naming convention.

### Review checklist

- [X] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [X] Feature or bugfix has tests
- [X] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by ChristineTChen — APPROVED** [review]

File: cert_manager/datadog_checks/cert_manager/metrics.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 METRIC_MAP = {
     'certmanager_certificate_ready_status': 'certificate.ready_status',
     'certmanager_certificate_expiration_timestamp_seconds': 'certificate.expiration_timestamp',
-    'certmanager_acme_client_request_count': 'acme_client.request.count',
-    'certmanager_acme_client_request_duration_seconds': 'acme_client.request.duration',
+    'certmanager_http_acme_client_request_count': 'http_acme_client.request.count',
+    'certmanager_http_acme_client_request_duration_seconds': 'http_acme_client.request.duration',
```
Comment (hithwen) [inline cert_manager/datadog_checks/cert_manager/metrics.py:9]: Can someone be running an older version and run into issues?

File: cert_manager/metadata.csv [hunk 2]
```diff
@@ -2,8 +2,8 @@ metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation
...
-cert_manager.acme_client.request.duration.sum,gauge,,,,The sum of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.sum
-cert_manager.acme_client.request.duration.count,gauge,,,,The count of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.cnt
-cert_manager.acme_client.request.duration.quantile,gauge,,,,The quantiles of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.quant
+cert_manager.http_acme_client.request.count,count,,,,The number of requests made by the ACME client,1,cert-manager,cm.acme_req_cnt
+cert_manager.http_acme_client.request.duration.sum,gauge,,,,The sum of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.sum
+cert_manager.http_acme_client.request.duration.count,gauge,,,,The count of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.cnt
+cert_manager.http_acme_client.request.duration.quantile,gauge,,,,The quantiles of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.quant
```
Comment (hithwen) [inline cert_manager/metadata.csv:8]: This is a breaking change, when releasing it should be a major bump

**Comment by hithwen** [comment]
Note that a PR bumping the version is needed to actually releas the changes

File: cert_manager/metadata.csv (hunk 2 above)
Comment (arapulido) [reply cert_manager/metadata.csv:8]: 👍

**Comment by arapulido** [comment]
> Note that a PR bumping the version is needed to actually releas the changes

Sounds good. I am preparing some more changes to this integration. Once those are done, I will create a new major release

**Pull request merged**